conf.d/hydro.fish

```diff
@@ -31,6 +31,39 @@ function _hydro_pwd --on-variable PWD --on-variable hydro_ignored_git_paths --on
     )
 end
 
+function _hydro_who
+    set --local show_hostname false
+    if set --query SSH_CONNECTION
+        set show_hostname true
+    else
+        switch (uname)
+        case Linux
+            if test -f /proc/1/environ && grep -qa container=lxc /proc/1/environ
```

Comment:
```suggestion
            if test -r /proc/1/environ && grep -qa container=lxc /proc/1/environ
```

I tested your branch since I want this feature, and got the following output:

```
Installing thenktor/hydro
           /home/ryan/.config/fish/functions/fish_mode_prompt.fish
           /home/ryan/.config/fish/functions/fish_prompt.fish
           /home/ryan/.config/fish/conf.d/hydro.fish
Installed 1 plugin/s
grep: /proc/1/environ: Permission denied
```

Using `test -r` instead of `test -f` will return false if the file doesn't exist or the user does not have read access.